test/built-ins/Temporal/Instant/prototype/toZonedDateTimeISO/timezone-wrong-type.js

```diff
@@ -27,6 +26,8 @@ for (const [timeZone, description] of rangeErrorTests) {
 
 const typeErrorTests = [
   [Symbol(), "symbol"],
+  [{}, "object not implementing time zone protocol"],
```

Comment:
Should there be a `null` case here too? I'm not adding the same comment to all the other tests like this, if that's OK.

Reply:
`null` is tested above in the list of RangeError cases - it falls through to the non-Object case in [ToTemporalTimeZoneSlotValue](file:///var/home/ptomato/workspace/proposal-temporal/out/index.html#sec-temporal-totemporaltimezoneslotvalue). (The Type() operation in the spec, thankfully, isn't the same as `typeof` in JS code — Type(**null**) is Null, not Object.)

Reply:
Whoops, sorry for not looking up. :-)